engines/scumm/gfx_gui.cpp

```diff
@@ -1666,31 +1652,38 @@ static const char *loomDraftsNames[6][17] = {
 	// HEBREW
 	{
 		"\x9a\x85\x90\x89\x82\x90\x8e",
```

Comment:
FWIW, the Hebrew translation to "Unknown" is "לא ידוע" and the string for it should be:
`":\x84\x85\x83\x89 \x80\x8c"`

Reply:
Thank you, I think I already have it as the last element of the array (I commited it yesterday), and this is how it looks:
![image](https://github.com/scummvm/scummvm/assets/11290886/96cdf02e-360b-41a4-8c24-c0b9f5abfb41)

Reply:
> Thank you, I think I already have it as the last element of the array (I commited it yesterday), and this is how it looks: ![image](https://user-images.githubusercontent.com/11290886/278078553-96cdf02e-360b-41a4-8c24-c0b9f5abfb41.png)

Looks great!